src/styles/product.scss

```diff
@@ -0,0 +1,93 @@
+:root {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  line-height: 140%;
+}
+
+body,
+p {
```

Comment:
The box-sizing property should be set to border-box globally, not only for body and p elements. Consider using the following selector: '*, *::before, *::after { box-sizing: border-box; }'